functions/.nvmrc

```diff
@@ -1 +0,0 @@
-v10.22.1
```

Comment:
I guess you thought of having a single `.nvmrc` file for this project, which is a nice idea. Although, since the front is using node `v10.17.0` (little difference), only removing this file causes a `regression` in the node version used for functions

Reply:
yes, we don't care much though :D

Reply:
which cause which issue?